src/KM_Game.pas

```diff
@@ -140,9 +140,10 @@   TKMGame = class
     property IsPaused: Boolean read fIsPaused write fIsPaused;
     property MissionMode: TKMissionMode read fMissionMode write fMissionMode;
     function GetNewUID: Integer;
-    function GetDefaultMPGameSpeed: Single;
-    procedure SetDefaultMPGameSpeed(aToggle: Boolean = False);
-    procedure SetGameSpeed(aSpeed: Single; aToggle: Boolean; aToggleSpeed: Single = 1);
+    function GetNormalGameSpeed: Single;
+    function GetNormalMPGameSpeed: Single;
+    procedure SetNormalMPGameSpeed(aToggle: Boolean = False);
+    procedure SetGameSpeed(aSpeed: Single; aToggle: Boolean; aNormalSpeed: Single = 1);
```

Comment:
This is a lot of functions for handling game speed, lets simplify this:
1. Remove GetNormalMPGameSpeed. GetNormalGameSpeed will return MP speeds if it's MP, or 1 if it's SP.
2. Remove aNormalSpeed parameter. We can assume it's always GetNormalGameSpeed.
3. Remove SetNormalMPGameSpeed. It's only used in MultiplayerRig so lets just move those 4 lines there.

After that we should just have:
function GetNormalGameSpeed: Single; (only used inside TKMGame.SetGameSpeed now, but might be useful elsewhere later on)
procedure SetGameSpeed(aSpeed: Single; aToggle: Boolean);

Calls to SetGameSpeed are simpler because they don't care about what is the "normal" speed. That's TKMGame's responsibility.